<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spatialPlotDialog</class>
 <widget class="QDialog" name="spatialPlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spatial Plots</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>347</y>
     <width>545</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>13</y>
     <width>521</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>10</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Spatial Feature Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>242</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Model Output File:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>231</y>
      <width>107</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFileName">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>36</y>
      <width>491</width>
      <height>181</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Element Features</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>80</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Spatial Average End Time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>9</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Plot Variable:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>48</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Spatial Average Start Time:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>376</x>
        <y>104</y>
        <width>107</width>
        <height>43</height>
       </rect>
      </property>
      <layout class="QHBoxLayout">
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEleShape">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="comboBoxEleVariable">
      <property name="geometry">
       <rect>
        <x>187</x>
        <y>4</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Interception Storage</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Saturated State</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Unsaturated State</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Surface State</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Evaporation from Canopy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Transpiration</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Evaporation from Ground/Surf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Precipitation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Net Precipitation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Infiltration</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Groundwater Recharge</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>116</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TIN Shape File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEleStart">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>44</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string comment="Enter Feature ID">Start Time</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEleFinish">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>75</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string comment="Enter Feature ID">End Time</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEleShape">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>113</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string comment="Enter Feature ID">Enter Time-Average Interval</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxEle">
      <property name="geometry">
       <rect>
        <x>366</x>
        <y>40</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Minute</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hour</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Days</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEditBinsEle">
      <property name="geometry">
       <rect>
        <x>422</x>
        <y>75</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>313</x>
        <y>47</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Units</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>313</x>
        <y>77</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Bins</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Stream Features</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>47</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Spatial Average Start Time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>79</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Spatial Average End Time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>9</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Plot Variable:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>376</x>
        <y>104</y>
        <width>107</width>
        <height>43</height>
       </rect>
      </property>
      <layout class="QHBoxLayout">
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRivShape">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineEditRivFinish">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>76</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string comment="Enter Feature ID">End Time</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRivVariable">
      <property name="geometry">
       <rect>
        <x>187</x>
        <y>4</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Head</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inflow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Outflow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Base Flow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Surface Flow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Base Flow Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Base Flow Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Surface Flow Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Surface Flow Right</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEditRivShape">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>113</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string comment="Enter Feature ID">Enter Time-Average Interval</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRivStart">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>45</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string comment="Enter Feature ID">Start Time</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>116</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>River Shape File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditBinsRiv">
      <property name="geometry">
       <rect>
        <x>421</x>
        <y>75</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>312</x>
        <y>77</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Bins</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>311</x>
        <y>47</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Units</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRiv">
      <property name="geometry">
       <rect>
        <x>366</x>
        <y>40</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Minute</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hour</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Days</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEditFileName">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>240</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>303</y>
     <width>531</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <widget class="QPushButton" name="pushButtonHelp">
      <property name="text">
       <string>  Help  </string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>  Close  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonGenerate">
      <property name="text">
       <string> Generate Shape </string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>spatialPlotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
